contracts/Obligation.sol

```diff
@@ -244,16 +244,13 @@ contract Obligation is IObligation, IERC20, European, Ownable, WriterRegistry {
 
         bytes32 id = keccak256(
             abi.encodePacked(
-                expiryTime,
-                windowSize,
-                strikePrice,
-                buyer,
-                seller,
-                // append nonce to avoid
-                // id collisions
+                // create2 ensures params are unique
+                address(this),
+                // append nonce to avoid collisions
                 _nonce
```

Comment:
note that we still have a 'global' nonce per obligation contract to distinguish requests on the same option